src/overset/AssembleOversetDecoupledAlgorithm.C

```diff
@@ -40,6 +40,8 @@ void AssembleOversetDecoupledAlgorithm::execute()
     fringeNodes.size(), KOKKOS_LAMBDA(const size_t& i) {
       coeffApplier->resetRows(1, &fringeNodes(i), 0, numDof, 1.0, 0.0);
     });
+
+  eqSystem_->linsys_->free_coeff_applier(coeffApplier);
```

Comment:
We're doing this again. We must have been leaking memory?

Reply:
Same reason as the above.